service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/usecase/TripManagementsUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.service_taxi.domain.usecase
+
+import org.thechance.service_taxi.domain.entity.Trip
+
+interface TripManagementsUseCase {
```

Comment:
shouldn't we add here the `delete `and `add `crud operations for a single trip here?

Reply:
add trip is responsibility of client not driver so should be here , but delete Hmm🤔🤔